apps/prairielearn/src/server.js

```diff
@@ -510,7 +510,8 @@ export async function initExpress() {
     );
   }
 
-  app.use('/pl/lti', (await import('./pages/authCallbackLti/authCallbackLti.js')).default);
+  app.use((await import('./middlewares/content-security-policy.js')).default);
```

Comment:
I reviewed the pages between line 486 where CSP used to be loaded to here, right before the first user-facing HTML page. Most of the pages are auth callbacks that do not have a UI. Two exceptions are the SAML test page and the LTI 1.3 test page, but they are simple non-interactive HTML without menus or anything at risk for clickjacking. 

The only concern I saw was in the LTI 1.3 router which includes the auth (which we want before CSP) and the `course_navigation` page which has instructor UI elements. I'll see if I can break that out of this router and into a page that loads later.

Reply:
Moved in db9bb0a3b